gatsby-theme-landing-page/src/components/divider.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import * as styles from "./divider.module.css";
+
+const dividerSize = {
+  S: styles.short,
```

Comment:
I'd suggest spelling out prop enum values; to me, S/M/L typically means "Small/Medium/Large"